CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## 4.1.2
```

Comment:
I just noticed that `4.1.1` isn't published to pub yet - should we roll these changes in with that an publish?